L/LaMEM/build_tarballs.jl

```diff
@@ -6,16 +6,16 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "LaMEM"
-version = v"2.1.2"
+version = v"2.1.3"
 
 
-PETSc_COMPAT_VERSION = "3.18.6" # Note: this is the version of the PETSc_jll package, which is sometimes larger than the PETSc version  
+PETSc_COMPAT_VERSION = "3.18.8" # Note: this is the version of the PETSc_jll package, which is sometimes larger than the PETSc version  
```

Comment:
This compat line will give it a compat entry of [3.18.8 - 4.0.0) I believe, but I think you said that PETSc changes ABI every release, so 3.19 won't be compatible with a build using 3.18. I think it would be better then to make this `~3.18.8` because that will limit it to only PETSc 3.18, and not try to automatically install PETSc 3.20 when you update to that.

Reply:
yes good point